CHANGELOG.md

```diff
@@ -1,3 +1,71 @@
+## 2023-06-14 v0.18.0
+
+> Note that there is a new naming scheme for release artifacts.
+
+## What's Changed
+
+Release highlights:
```

Comment:
Can we include that rln-relay uses atomic operations now? This is a milestone :)